lazy_tensor_core/lazy_bench.py

```diff
@@ -440,11 +440,27 @@ def lazy_compute_experiment(args, experiment, results, benchmark, lazy_benchmark
     print(f"CIDEBUGOUTPUT,lazy_compute_experiment,{current_name},{current_device},{experiment},{args.test},{speedup:.4f},{pvalue:.2e},{args.warmup},{args.repeat},{warmup_time:.2f},{bench_time:.2f}")
     return (speedup, pvalue)
 
-def check_results(name, correct_result, lazy_result, device):
+# TODO(alanwaketan): The check_results family may need a proper re-design.
```

Comment:
the other way this might be approached is to make to_device handle tuples.  I'm not sure if torch.allclose works on tuples currently, if it does that would simplify things.

Reply:
I don't think so. It complains about the inputs are not torch.Tensor.